### PR TITLE
Add inline tabs on Git Bootcamp page for commands on different systems

### PR DESCRIPTION
**Pull request by lancegoyke**
This PR should close a single TODO in https://github.com/python/devguide/issues/1196.

----
:books: Documentation preview :books:: https://cpython-devguide--1225.org.readthedocs.build/getting-started/git-boot-camp/

### CODE REVIEW

File: getting-started/git-boot-camp.rst [hunk 1]
```diff
@@ -439,16 +439,25 @@ Both of these tools will configure a remote URL for the branch, so you can
 ``git push`` if the pull request author checked "Allow edits from maintainers"
 when creating the pull request.
 
-If you don't have GitHub CLI or hub installed, you can set up a git alias. On
-Unix and macOS::
+If you don't have GitHub CLI or hub installed, you can set up a git alias:
 
-   $ git config --global alias.pr '!sh -c "git fetch upstream pull/${1}/head:pr_${1} && git checkout pr_${1}" -'
+.. tab:: Unix
```
Comment (hugovk) [inline getting-started/git-boot-camp.rst:444]: I think we can combine these into a single tab:

```suggestion
.. tab:: Unix/macOS
```

File: getting-started/git-boot-camp.rst [hunk 2]
```diff
@@ -439,16 +439,25 @@ Both of these tools will configure a remote URL for the branch, so you can
...
 
-.. code-block:: bash
+      $ git config --global alias.pr '!sh -c "git fetch upstream pull/${1}/head:pr_${1} && git checkout pr_${1}" -'
+
+.. tab:: macOS
+
+   .. code-block:: shell
+
+      $ git config --global alias.pr '!sh -c "git fetch upstream pull/${1}/head:pr_${1} && git checkout pr_${1}" -'
+
```
Comment (hugovk) [inline getting-started/git-boot-camp.rst:455]:
```suggestion
```

**Review by hugovk — COMMENTED** [review]
Please also add the JS.

File: getting-started/git-boot-camp.rst (hunk 1 above)
Comment (lancegoyke) [reply getting-started/git-boot-camp.rst]: Makes sense to me!

**Review by hugovk — APPROVED** [review]
Thank you!

**Review by ezio-melotti — APPROVED** [review]

**Pull request merged**